lang/es_es.lang

```diff
@@ -36,8 +36,7 @@ redactor2_profiles_label_linkify = ¿Convertir enlaces automáticamente?
 redactor2_profiles_label_linkify_option_true = Sí
 redactor2_profiles_label_linkify_option_false = No
 redactor2_profiles_label_redactorplugins = Plugins
-redactor2_profiles_label_redactorcustomplugins = Complementos personalizados
-redactor2_profiles_label_redactorcustomplugins_help = Coma separada por la sintaxis "name:path/to/plugin.js"
+redactor2_profiles_label_redactorcustomplugins_help = Include your plugins via your project addon. (-EP PACKAGES_INCLUDED LATE-)
```

Comment:
frag mal gregor, ich glaube dass wir keine englischen keys in anderen sprachen machen. den key sollte man dann vermutlich leer machen, oder rauslöschen

Reply:
klingt sinnvoll.. ändere ich